spec/factories/disabilities.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
```

Comment:
Leave this for now but I wonder if we should use fixtures for all this seed type data. We don't and probably won't ever be adding functionality where we're mutating these things. We just need them to be around when the tests run.

Reply:
So fixtures for simple models but factories where there's a bit more involved with setting up test data?